ChangeLog

```diff
@@ -1,4 +1,8 @@
 2017-07-24  Leah South  <leah.south@hdr.qut.edu.au>
+
+    * inst/include/sampler.h: Correcting the description of GetParticleWeight().
+    
+2017-07-24  Leah South  <leah.south@hdr.qut.edu.au>
```

Comment:
Nitwick: when one has two entries _with the same data and author_ one tends to merge those.

But thanks for keeping up with ChangeLog!